v3/src/models/document/free-tile-row.ts

```diff
@@ -158,6 +158,14 @@ export const FreeTileRow = TileRowModel
     setTileDimensions(tileId: string, dimensions: { width?: number, height?: number }) {
       const freeTileLayout = self.getNode(tileId)
       freeTileLayout?.setSize(dimensions.width, dimensions.height)
+    },
+    setTilePosition(tileId: string, position: { x?: number, y?: number }) {
+      const freeTileLayout = self.getNode(tileId)
+      if (freeTileLayout) {
```

Comment:
```suggestion
      if (freeTileLayout && (x != null || y != null)) {
```
so we skip the whole thing if they're both nullish.